.github/workflows/daily-product-scan.yml

```diff
@@ -1,6 +1,7 @@
 on:
-  schedule:
-    - cron: '0 12 * * 1-5'
+  push
```

Comment:
Is this testing change that will be reverted afterwards?

Reply:
Thank for catching that, Theo. I just updated it.

Reply:
No worries, I just approved ✅